kong/pdk/client/tls.lua

```diff
@@ -45,22 +46,39 @@ local function new()
   -- To find out whether the client honored the request, use
   -- `get_full_client_certificate_chain` in later phases.
   --
+  -- The `ca_certs` argument is the CA certificate chain opaque pointer (if any),
+  -- which can be created by the [parse_pem_cert](https://github.com/openresty/lua-resty-core/blob/master/lib/ngx/ssl.md#parse_pem_cert)
+  -- or [resty.opensslx509.chain](https://github.com/fffonion/lua-resty-openssl#restyopensslx509chain)
+  -- The list of certificates will be sent to clients. Also, they will be added
+  -- to trusted store. If omitted, will not send any CA certificate to clients.
```

Comment:
Is the trusted store even used for our implementation? If not we should remove mentioning of it to avoid confusions.

Reply:
Good idea! Done.